ProjectCoimbra.UWP/Project.Coimbra/App.xaml.cs

```diff
@@ -28,6 +28,8 @@ public App()
             this.InitializeComponent();
             this.Suspending += this.OnSuspending;
             NetworkListener.StartListener();
+
+            DataAccess.InitializeDatabase();
```

Comment:
I think this needs to be `await`ed. You can use the `.Wait()` here as the method can't be made `async`.

Right now, this isn't right as there's no guarantee that the method will finish running ever.